modules/plugin/coverage-multidim/netcdf/src/main/java/org/geotools/coverage/io/netcdf/crs/NetCDFProjection.java

```diff
@@ -175,8 +176,11 @@ public CoordinateReferenceSystem parseCoordinateReferenceSystem(Variable var)
                     handleParam(paramsMapping, netcdfParameters, ogcParameterKey, var);
                 }
 
+                Map<String, Object> map = new HashMap<String, Object>();
+                map.put(NetCDFUtilities.NAME, projectionName);
```

Comment:
So the projection name could be overridden by the provided crsProperties?
If that does not make sense, it's easier to just pass the crsProperties to the hash map constructor and then put then name on top of it... or not?

Reply:
Right. At the beginning there was the case of a null crsProperties. I ended up using an emptyMap instead so it's now safe to have it in the constructor.
fixing.